tests/integration/conftest.py

```diff
@@ -327,3 +327,48 @@ def required_packages(shell):
 
     except KeyError:
         raise
+
+
+@pytest.fixture(scope="function")
+def repositories(shell):
```

Comment:
I would name it something like `backup_system_repositories` or something like that which is more self-explaining.

Reply:
Will update in separate PR to unblock merge